tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -195,6 +202,34 @@ public static function clearCacheAfterFeature()
         self::clearCache();
     }
 
+    /**
+     * @BeforeScenario @reset-context-before-scenario
```

Comment:
```suggestion
     * @BeforeScenario @reset-context-after-scenario
```
This name would be more adapted I guess